src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java

```diff
@@ -100,9 +99,10 @@ private Flux<WebSocketMessage> notificationFlux(WebSocketSession webSocketSessio
                                                     String filterUpdateType) {
         return flux.transform(f -> {
             Flux<Message<String>> res = f;
-            if (userId != null) {
-                res = res.filter(m -> m.getHeaders().get(HEADER_ERROR) == null || userId.equals(m.getHeaders().get(HEADER_USER_ID)));
-            }
+            // code commented (waiting for refactor) to be able to receive the notification even when there is an error, then the front choose to display it or not
```

Comment:
Usually, we don't keep commented code, even if we think we'll use it later. 
It's benefic to keep there ?

Reply:
Ok I'll remove it

Reply:
Done